dev/jobs/prep.sh

```diff
@@ -129,23 +142,25 @@ if [[ ${MAKE_PREPBUFR} = "YES" ]]; then
     # Do not fail on external errors
     set +eu
     "${HOMEobsproc}/jobs/JOBSPROC_GLOBAL_PREP" && true
-    err=$?
+    export err=$?
     if [[ ${err} -ne 0 ]]; then
-       echo "FATAL ERROR: Global prep job failed!"
-       exit 1
+       err_exit "JOBSPROC_GLOBAL_PREP job failed!"
     fi
-    set_strict
 
     # If creating NSSTBUFR was disabled, copy from DMPDIR if appropriate.
     if [[ ${MAKE_NSSTBUFR:-"NO"} = "NO" ]]; then
-        if [[ ${DONST} = "YES" ]]; then ${NCP} "${COMIN_OBSDMP}/${OPREFIX}nsstbufr" "${COMOUT_OBS}/${OPREFIX}nsstbufr"; fi
+        if [[ ${DONST} = "YES" ]]; then
+           cpfs "${COMINobsproc}/${OPREFIX}nsstbufr" "${COMOUT_OBS}/${OPREFIX}nsstbufr"
+        fi
     fi
 
 else
     if [[ ${ROTDIR_DUMP} = "NO" ]]; then
-        ${NCP} "${COMIN_OBSDMP}/${OPREFIX}prepbufr"               "${COMOUT_OBS}/${OPREFIX}prepbufr"
-        ${NCP} "${COMIN_OBSDMP}/${OPREFIX}prepbufr.acft_profiles" "${COMOUT_OBS}/${OPREFIX}prepbufr.acft_profiles"
-        if [[ ${DONST} = "YES" ]]; then ${NCP} "${COMIN_OBSDMP}/${OPREFIX}nsstbufr" "${COMOUT_OBS}/${OPREFIX}nsstbufr"; fi
+        cpfs "${COMINobsproc}/${OPREFIX}prepbufr" "${COMOUT_OBS}/${OPREFIX}prepbufr"
+        cpfs "${COMINobsproc}/${OPREFIX}prepbufr.acft_profiles" "${COMOUT_OBS}/${OPREFIX}prepbufr.acft_profiles"
+        if [[ ${DONST} = "YES" ]]; then
+           cpfs "${COMINobsproc}/${OPREFIX}nsstbufr" "${COMOUT_OBS}/${OPREFIX}nsstbufr"
+        fi
```

Comment:
I explained why I recommended `cpreq` here.
These files are required in the `anal.sh` step of the GSI.  So, if they are not present here, then the GSI will fail in a later job.  Having a `cpreq` here allows us to trap that error much sooner.